content/nginx-one/secure-your-fleet/secure.md

```diff
@@ -0,0 +1,131 @@
+---
+title: "Set up security alerts"
+weight: 500
+toc: true
+nd-content-type: how-to
+nd-product: NGINX One
+---
+
+With this page, you'll learn how to set up alerts in F5 Distributed Cloud. Once configured, you'll see the CVEs and insecure configurations associated with your NGINX fleet. These instructions are intended for those responsible for keeping their NGINX infrastructure and application traffic secure. It assumes you know how to:
+
+- Install Linux programs or run Docker containers
+
+By the end of this tutorial, you'll be able to:
+
+- Access the NGINX One Console in F5 Distributed Cloud
+- Connect NGINX instances to the NGINX One Console
+- Review Security Risks associated with your NGINX fleet
+- Configure Alert Policies in F5 Distributed Cloud
+
+## Background
+
+NGINX One Console is a service to monitor and manage NGINX. It's a part of the F5 Distributed Cloud and is included with all NGINX and F5 Distributed Cloud subscriptions. While NGINX is built to be secure and stable, critical vulnerabilities can occasionally emerge – and misconfigurations may leave your applications or APIs exposed to attacks. 
+
+## Before you begin
+
+If you already have accessed F5 Distributed Cloud and have NGINX instances available, you can skip these steps and start to connect instances to the NGINX One Console.
+
+### Confirm access to the F5 Distributed Cloud
+
+Confirm an F5 Distributed Cloud tenant has been provisioned for you. Log in to MyF5 and review your subscriptions. You should see within one of your subscriptions "Distributed Cloud". This could be in either an NGINX subscription or a Distributed Cloud. If the above does not appear in any of your subscriptions, reach out to either your F5 Account Team or Customer Success Manager.
+
+With access, you or someone in your organization should have an email from no-reply@cloud.f5.com asking you to update your password when the tenant was created. The account name referenced in the E-Mail in bold is the tenant name.
+
+Navigate to https://INSERT_YOUR_TENANT_NAME.console.ves.volterra.io/ to access F5 Distributed Cloud. If you have never logged in, select the **Forgot Password?** option in the log in screen. Alternatively, if someone within your organization has access, ask them to add you as a user within your tenant with a role providing permissions for NGINX One.
```

Comment:
```suggestion
Go to https://INSERT_YOUR_TENANT_NAME.console.ves.volterra.io/ to access F5 Distributed Cloud. If you have never logged in, select the **Forgot Password?** option in the log in screen. Alternatively, if someone within your organization has access, ask them to add you as a user within your tenant with a role providing permissions for NGINX One.
```

Reply:
Style guide says to use "Go to" instead of "Navigate."

https://github.com/nginx/documentation/blob/main/documentation/style-guide.md

(Aside: I think navigate harkens back to the Netscape Navigator days -- when the Web was like an ocean. Netscape's early logo was a ship's wheel.)